index.html

```diff
@@ -2517,26 +2518,68 @@ <h3>
         <dl>
           <dt>@context</dt>
           <dd>
-The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
-where the value of the first <a>URI</a> is
-<code>https://www.w3.org/ns/did/v1</code>. All members of the
+The value of the <code>@context</code> property MUST be exactly one of the following:
+
+<ul>
+  <li>The string <code>https://www.w3.org/ns/did/v1</code>. 
+    <pre class="example">
+      {
+        "@context": "https://www.w3.org/ns/did/v1",
+        "id": "did:example:123"
+      }      
+    </pre>
+  </li>
+  <li>An array of one or more elements,
+    where the value types of the members of the array
+    MUST be either a string or an object and the first element is
+    the string <code>https://www.w3.org/ns/did/v1</code>.
+
+    <pre class="example">
+      {
+        "@context": [
+          "https://www.w3.org/ns/did/v1",
+          "https://example.com/blockchain-identity/v1"
+        ],
+        "id": "did:example:123"
+      }      
+    </pre>
+  </li>
+  <li>An array where the first value is
+    <code>https://www.w3.org/ns/did/v1</code>, and the final value is an object.
+
+    <pre class="example">
+      {
+        "@context": [
+          "https://www.w3.org/ns/did/v1",
+          "https://example.com/blockchain-identity/v1",
+          {
+            "@base": "did:example:123"
+          }
```

Comment:
Feeling stronger about this now... let's not invite trouble... can we just say something to the effect of "anything that is valid for JSON-LD is valid here" and then give a simple example that defines a one-type use property (even that I'm feeling uncomfortable stating).

Reply:
@msporny "anything that is valid for JSON-LD is valid here" this is exactly what I am trying to say! :)

Can you propose concrete language here? We don't need to even define `@context` if we say that we support `100%` of JSON-LD, and just link to that defintion.... I would prefer to just say that, and provide an example that uses JSON-LD relative refs, in the JSON-LD representation section.

Reply:
@msporny I attempted to clarify this in https://github.com/w3c/did-core/pull/408/commits/f9f1ee80144cc434533b530c23c66c83c33bcda9

Reply:
Not my style, but it's not harmful, and it seems like people want the extra verbosity. I'm standing aside.

Reply:
I still think `@base` examples in DID Core is a bad idea. I'm actively blocking this PR -- we need to get broader consensus from the group on this.